tephra-core/src/main/java/co/cask/tephra/util/TxUtils.java

```diff
@@ -30,6 +30,10 @@
  */
 public class TxUtils {
 
+  // Any cell with timestamp lesser than MAX_NON_TX_TIMESTAMP is assumed to be pre-existing data,
+  // i.e. data written before table was converted into transactional table using Tephra.
+  private static final long MAX_NON_TX_TIMESTAMP = (long) (System.currentTimeMillis() * 1.1);
```

Comment:
It is okay for this constant to change every run. We just need a timestamp in milliseconds range to check whether a cell's timestamp is transational or not. Since transactional timestamps are 10⁶ times a timestamp in milliseconds, it should not matter whether this time is off by a few seconds due to class loading.